c2corg_ui/static/js/protectedurlbtn.js

```diff
@@ -0,0 +1,61 @@
+goog.provide('app.protectedUrlBtnController');
```

Comment:
``protectedUrlBtnController`` => ``ProtectedUrlBtnController`` (capital initial letter for controllers)